### PR TITLE
1212 large filer cypress

### LINKED ISSUE #1212: Add cypress test for 1m row submission, and notify MM if not successful
Schedule this test to run on nightly basis

- [x] Generate 494,099 rows / 511.9MB row test file for Frontend Test Bank
- [x] Confirm test file can be submitted and signed
- [x] Create `Large Filer` Cypress test
- ~Create separate shell script to run `Large Filter` test separately? Create separate image tag?~
- [x] Update Cypress Large Filer testing image
- [x] Verify MM integration on test failure
- [x] Verify MM integration on success
- [x] Resolve v716 in 2022 test file
- [x] Discuss implications of this testing
  - [x] Pushing >1GB of filing data daily, but it's not needed long-term.  Should we automate its cleanup?
- [x] Decouple max-test-file from the test-image?
  - [x] 500MB file is too large to store on Github
  - [x] Store it in S3 and `curl` it in the `docker-runner` script?

### PR DESCRIPTION
**Pull request by meissadia**
Closes #1212 
## Changes
- LAR file generation script
- Adds Cypress load-test for 2020
- Copies load-test LAR file from S3 during test image build, if needed.

## Testing
Load-testing has been successfully running against Prod since early February 👍🏽

### CODE REVIEW

File: cypress/Dockerfile [hunk 1]
```diff
@@ -44,4 +44,7 @@ RUN useradd -ms /bin/bash hmda_cypress_user && \
 
 USER hmda_cypress_user
 
-RUN yarn cypress install
\ No newline at end of file
+RUN yarn cypress install
+
+# Copy load-test file from S3 if not already present
```
Comment (meissadia) [inline cypress/Dockerfile:49]: My original plan was to perform this copy [inside of the running pod](https://github.com/cfpb/hmda-frontend/pull/1330/files#diff-d92cde168d4ccc1a2eff895bf45b8573cd02f7c0ec4580c62f58edb4c999a589R55), however I was unable to set the correct file permissions on `s3://cfpb-hmda-public/cypress/2020-FRONTENDTESTBANK9999-MAX.txt` to make it publicly accessible via `curl`, hence this approach.  
 
[Access Denied](https://s3.amazonaws.com/cfpb-hmda-public/cypress/2020-FRONTENDTESTBANK9999-MAX.txt); only downloadable with `aws` cli, using the`svc_hmda` profile.

Comment (lchen-2101) [reply cypress/Dockerfile:49]: did you create this file through the aws web console? If this file should be / could be made public, you can do so in the console, there's a `Make public using ACL` selection you can follow.

File: cypress/integration/load/LargeFiler.spec.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  TEST_DELAY,
+  ENVIRONMENT,
+  AUTH_BASE_URL,
+  AUTH_REALM,
+  AUTH_CLIENT_ID,
+  YEARS,
+} = Cypress.env()
+
+const years = [2020]
+const timeout3hours = 10800000
```
Comment (lchen-2101) [inline cypress/integration/load/LargeFiler.spec.js:19]: think it'd be easier to read if we did `3 * 60 * 60 * 1000`, so if we wanted to change anything later, it'd be more easily understandable.

Comment (meissadia) [reply cypress/integration/load/LargeFiler.spec.js]: I like it 👍🏽

**Review by lchen-2101 — APPROVED** [review]
LGTM, look into the S3 placement later for public access

**Pull request merged**